rpcserver.go

```diff
@@ -6464,20 +6464,21 @@ func (r *rpcServer) GetTransactions(ctx context.Context,
 	}
 
 	// To remain backward compatible, If the number of transactions was not
-	// specified, then we'll default to returning the entire transactions
+	// specified, then we'll default to returning the entire transactions.
 	if req.MaxTransactions == 0 {
 		req.MaxTransactions = -1
 	}
 
-	transactions, err := r.server.cc.Wallet.ListTransactionDetails(
-		req.StartHeight, endHeight, req.Account, int(req.IndexOffset),
-		int(req.MaxTransactions),
-	)
+	transactions, firstIdx, lastIdx, err :=
```

Comment:
nit: same here re `transactions -> txns` to optimize formatting.

Reply:
I replaced "transactions" with "txns," but the line is still too long.